src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -810,7 +810,10 @@ static bool ValidateConcatAxis(const InferenceEngine::CNNLayerPtr layer, std::st
         if (unsupported_concat_axis != end_dim) {
             auto dims = concat_layer->insData[0].lock()->getDims();
             std::ostringstream in_dims_oss;
-            std::copy(dims.begin(), dims.end(), std::ostream_iterator<size_t>(in_dims_oss, ","));
+            std::copy(dims.begin(), std::prev(dims.end()), std::ostream_iterator<size_t>(in_dims_oss, ","));
+            if (!dims.empty()) {
+                std::cout << dims.back();
```

Comment:
Why do you print here dirrectly to stdout instead of in_dims_oss

Reply:
By mistake :-) Will fix.